inter-server-sync.spec

```diff
@@ -40,9 +40,9 @@ Source0:        %{name}-%{version}.tar.gz
 Source1:        vendor.tar.gz
 BuildRequires:  golang-packaging
 %if 0%{?rhel}
-BuildRequires:  golang >= 1.16
+BuildRequires:  golang >= 1.18
```

Comment:
In the changelog you mentioned Go 1.19 as minimum requirement but on RHEL you are bumping it only to 1.18. is this intended?

Reply:
I've seen that some RH projects in OBS/IBS were missing Go 1.19.

Reply:
Done